README.md

```diff
@@ -1,20 +1,92 @@
-# osl-repos
+osl-repos
+================
 
-TODO: Enter the cookbook description here.
+This cookbook manages the base, epel, extras, and updates repositories on Centos 7; and the appstream, base, elrepo,
+epel, and powertools repositories on Centos 8. This cookbook imports these repositories and points them at the osuosl
+repositories.
 
-## Requirements
+## Repository
 
-### Platforms
-
-- CentOS 7+
+```
+https://github.com/osuosl-cookbooks/osl-repos
+```
 
-### Cookbooks
+## Recipes
 
-## Attributes
+- `osl-repos::centos`  - Configures and enables all repositories, with the exception of the 'elrepo' and 'epel' repos
+- `osl-repos::elrepo`  - Configures and enables the 'elrepo' repository
+- `osl-repos::epel`    - Configures and enables the 'epel' repository
 
 ## Resources
 
-## Recipes
+- `osl_repos_centos` - Manages and configures the appstream, base, extras, highavailability, powertools, and updates repositories using the `yum-centos` cookbook
+- `osl_repos_elrepo` - Manages and configures the elrepo repository using the `yum-epel` cookbook
+- `osl_repos_epel`   - Manages and configures the epel repository using the `yum-elrepo` cookbook
+
+### Actions:
+
+| Action 	| Description                                                            	|
+|--------	|------------------------------------------------------------------------	|
+| add    	| Configures all repositories managed by a given resource                	|
+
+### Properties:
+
+Note: All repositories controlled by a resource will be installed and configured. These properties determine if said repos are **enabled**
+
+| Resource  	| Property        	| Effect                     	      | Default  | Compatibility     |
+|-----------	|-----------------  |---------------------------------	|--------  |-----------------  |
+| centos 	    | appstream     	  | Enable the appstream repo   	    | True     | Centos 8          |
+| centos 	    | base           	  | Enable the base repo   	          | True     | Centos 7 and 8    |
+| centos      | extras            | Enable the extras repo   	        | True     | Centos 7 and 8    |
+| centos   	  | highavailability  | Enable the highavailability repo  | False    | Centos 8          |
+| centos   	  | powertools        | Enable the powertools repo   	    | True     | Centos 8          |
+| centos      | updates        	  | Enable the updates repo   	  	  | True     | Centos 7          |
+| epel    	  | epel           	  | Enable the epel repo   	  	      | True     | Centos 7 and 8    |
+| elrepo  	  | elrepo         	  | Enable the elrepo repo   	  	    | True     | Centos 7 and 8    |
+
+
+## Examples:
+
+Configure the default suite of Centos repos by recipe:
+```ruby
+include_recipe 'osl-repos::centos'
+```
+Or resource:
+```ruby
+osl_repos_centos 'default'
+```
```

Comment:
@ramereth Should we only recommend using the resource?

I recommend collapsing both examples into a single codeblock for better grouping. Please do this for the other examples as well.
````suggestion
Configure the default suite of Centos repos:
```ruby
# with the recipe:
include_recipe 'osl-repos::centos'

# with a resource:
osl_repos_centos 'default'
```
````